Cargo.toml

```diff
@@ -10,4 +10,16 @@ members = [
 # chainblocks = { path = "../chainblocks/rust" }
 
 [profile.release]
-panic = 'unwind'
+# Substrate runtime requires unwinding.
+panic = "unwind"
+
+# Inspired from https://github.com/paritytech/substrate/blob/master/Cargo.toml:
+[profile.production]
+inherits = "release"
+
+# Sacrifice compile speed for execution speed by using optimization flags:
+
+# https://doc.rust-lang.org/rustc/linker-plugin-lto.html
+lto = "fat"
```

Comment:
why? makes it very slow to link and we don't need such optimization?

Reply:
> The production profile applies various compiler optimizations.
These optimizations slow down the compilation process a lot.
If you are just testing things out and don't need final numbers, use the --release command-line option instead of the production profile. (https://docs.substrate.io/test/benchmark/)

This is only for testing things out